modules/boost.detail/1.83.0/MODULE.bazel

```diff
@@ -0,0 +1,13 @@
+module(
+    name = "boost.detail",
+    version = "1.83.0",
```

Comment:
We should add here `bazel_compatibility = [">=7.2.1"],` since overlays work only with Bazel 7.2.1 and up